server/PathServices/SignalRRealtimeDataRepository.cs

```diff
@@ -16,10 +17,11 @@
     /// </summary>
     internal sealed class SignalRRealtimeDataRepository : IRealtimeDataRepository, IDisposable
     {
+        private readonly TimeSpan KEEP_ALIVE_INTERVAL = TimeSpan.FromTicks(5 * (long)10e6); // 5s
```

Comment:
Bit of an off way to define this.

TimeSpan.FromSeconds(5) would be much easier to read.